src/app/feed/feed-advanced-search/feed-advanced-search.component.html

```diff
@@ -0,0 +1,11 @@
+	<div class="tabslist">
+		<ul>
+			<li [class.active_view]="show('all')" class="tabs" (click)="getAllResults()">All</li>
```

Comment:
Don't make this show method, you can handle this logic inline only `[class.active]="selectedTab==='all'"` no need to make this function call.